src/recon_buildblock/BackProjectorByBinUsingProjMatrixByBin.cxx

```diff
@@ -237,5 +241,41 @@ actual_back_project(DiscretisedDensity<3,float>& image,
 }
 
 
+void
+BackProjectorByBinUsingProjMatrixByBin::
+actual_back_project(DiscretisedDensity<3,float>& image,
+                    const Bin& bin)
+{
+    if (proj_matrix_ptr->is_cache_enabled() /*&& !tof_enabled*/)
+    {
+        ProjMatrixElemsForOneBin proj_matrix_row;
+        proj_matrix_ptr->get_proj_matrix_elems_for_one_bin(proj_matrix_row, bin);
+        proj_matrix_row.back_project(image, bin);
+    }
+    else if (proj_matrix_ptr->is_cache_enabled()/* && tof_enabled*/)
+    {
+        tof_row->back_project(image, bin);
+    }
```

Comment:
this should be reviewed. It is likely obsolete. If we do need it, the interface with `tof_row` etc should be made a lot cleaner (no bare pointers!)